packages/wallet/test/services/TransactionsTracker.test.ts

```diff
@@ -338,7 +338,9 @@ describe('TransactionsTracker', () => {
         const failedToSubmit$ = hot<FailedTx>('---a|', {
           a: { reason: TransactionFailure.FailedToSubmit, ...outgoingTx }
         });
-        const failedFromReemitter$ = cold<FailedTx>('-a|', { a: { reason: TransactionFailure.Timeout, ...outgoingTxReemit } });
+        const failedFromReemitter$ = cold<FailedTx>('-a|', {
```

Comment:
It's surprising the lint rules allow either of these options (the before and now) :thinking: Seems like we're not enforcing some rules in tests?

Reply:
The rules are disabled for the whole file. Probably to allow the lengthy test names.

Reply:
If I remember right it was disabled for more readable test setup:
```
cold('------a----', { a: 'some long object definition' });
cold('-------a---');
```
vs
```
cold('------a----', {
  a: 'some long object definition'
});
cold('-------a---');
```
You can understand the setup easier when there are many of those aligned one after another. Even when you use a variable of this long object definition, the names are sometimes too long or there are simply too many of emitted items.

Reply:
OK, it makes sense, I'm just expecting the lint rules to be enforcing style. I wonder if there's one to favour this automatically, but it's not important now.

Reply:
Unfortunately, it was lost at some point, probably due to editors like mine with prettier formatter enabled, which ignores the eslint file rules.

Reply:
Agree @rhyslbw, we should probably just have different rules for 'test/' to not have to eslint-disable as much. We do that a lot in tests. `any` is typically acceptable for example.